client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,155 @@
+  );
+  const [selectedBooking, setSelectedBooking] = useState<BookingRequest | undefined>(nextBooking);
+
+  if (loggedInUser === undefined) return <CircularProgress />;
+  if (!loggedInUser) {
+    history.push('/login');
+    // loading for a split seconds until history.push works
+    return <CircularProgress />;
+  }
+
+  const handleDateChange = (e: MaterialUiPickersDate) => {
+    if (!e) return;
+    changeDate(e);
+    setIsSelectedDate(true);
+    const request = requests.find(
+      (request) => Intl.DateTimeFormat('en').format(request.start) === Intl.DateTimeFormat('en').format(e),
+    );
+    setSelectedBooking(request);
+  };
+
+  const renderDay = (
+    day: MaterialUiPickersDate,
+    selectedDate: MaterialUiPickersDate,
+    dayInCurrentMonth: boolean,
+    dayCompmonent: JSX.Element,
+  ) => {
+    const component =
+      day &&
+      Intl.DateTimeFormat('en').format(nextBooking.start) ===
+        Intl.DateTimeFormat('en').format(new Date(day.toString())) ? (
+        <Avatar className={classes.upcomingBookingCircle}>{dayCompmonent}</Avatar>
+      ) : (
+        <span>{dayCompmonent}</span>
+      );
+    return component;
+  };
+
+  return (
+    <Grid container className={classes.root}>
+      <Grid container xs={6} className={classes.bookingsPart}>
+        <Paper key="selectedBookingPaper" className={classes.bookingPaper}>
+          <Typography className={classes.selectedBooking}>
+            {isSelectedDate ? 'BOOKINGS ON SELECTED DATE:' : 'YOUR NEXT BOOKING:'}
+          </Typography>
+          {(selectedBooking && (
+            <Booking
+              status={selectedBooking.status}
+              start={new Date(selectedBooking.start)}
+              end={new Date(selectedBooking.end)}
+              sitterId={selectedBooking.sitterId}
+            />
+          )) || (
+            <Paper>
+              <Grid className={classes.noBooking}>
+                <Typography>No bookings on selected date</Typography>
+              </Grid>
+            </Paper>
+          )}
+        </Paper>
+        <Paper key="currentBookingsPaper" className={classes.bookingPaper}>
+          <Typography className={classes.selectedBooking}>CURRENT BOOKINGS:</Typography>
+          <Box overflow="auto" style={{ height: '200px' }}>
+            {requests.map((request) => {
+              if (request.start > new Date() && request != selectedBooking) {
+                return (
+                  <Booking
+                    key={`${request.sitterId}${request.start.toString()}`}
+                    status={request.status}
+                    start={request.start}
+                    end={request.end}
+                    sitterId={request.sitterId}
+                  />
+                );
+              }
+            })}
+          </Box>
+        </Paper>
+        <Paper key="pastBookingsPaper" className={classes.bookingPaper}>
+          <Typography className={classes.selectedBooking}>PAST BOOKINGS:</Typography>
```

Comment:
Let's avoid using texts in all capital letters and instead utilize `CSS` to convert the text into uppercase. Here and anywhere else that applies 👍🏼